src/main/java/cubicchunks/asm/mixin/core/common/MixinWorld.java

```diff
@@ -219,16 +220,24 @@ public boolean testForCubes(CubePos start, CubePos end, Predicate<Cube> cubeAllo
     
     /**
      * @author Foghrye4
-     * @reason Original {@link World#markChunkDirty(BlockPos, TileEntity)} 
-     * called by TileEntities whenever they need to force Chunk to save 
-     * valuable info they changed.
-     * Because now we store TileEntities in Cubes instead of Chunks, it will
-     * be quite reasonable to force Cubes to save themselves.
+     * @reason Original {@link World#markChunkDirty(BlockPos, TileEntity)}
+     *         called by TileEntities whenever they need to force Chunk to save
+     *         valuable info they changed. Because now we store TileEntities in
+     *         Cubes instead of Chunks, it will be quite reasonable to force
+     *         Cubes to save themselves.
```

Comment:
These formatting changes are weird. Is this the result of applying the formatter?